optimism2/ovm2/get_contracts/insert_get_contracts.sql

```diff
@@ -174,7 +174,7 @@ GROUP BY 1,2,3,4,5,6
                   fifth_level -- check for contract factories 4 layers down --as of 3/12, this ran through all contracts
               ) con
               LEFT JOIN ovm2."contract_creator_address_list" cc ON con.creator_address = cc.creator_address
-              LEFT JOIN ovm2."contract_creator_address_list" ccf ON con.creator_address = ccf.contract_creator_if_factory
+              LEFT JOIN ovm2."contract_creator_address_list" ccf ON con.creator_address = ccf.creator_address
```

Comment:
Ah had a typo here, will fix. Shouldn't create any major impact for now, but I'll check. Can be fixed with an update if needed.